src/document/crdt/tree.ts

```diff
@@ -453,9 +453,9 @@ export class CRDTTreeNode extends IndexTreeNode<CRDTTreeNode> {
 }
 
 /**
- * toJSON converts the given CRDTNode to JSON.
+ * `toJS` converts the given CRDTTreeNode to TreeNode.
  */
-function toJSON(node: CRDTTreeNode): TreeNode {
+function toJS(node: CRDTTreeNode): TreeNode {
```

Comment:
"JS" is meaningless in Swift. How about changing the name to more specific?
I suggest "toTreeNode" as the name.

Reply:
I think using the specific term, as suggested above, is better as `JS` doesn't convey the intended meaning in kotlin either.